sign/sample/dapp/src/main/java/dev/pinkroom/walletconnectkit/sign/dapp/sample/main/DropdownMenu.kt

```diff
@@ -45,6 +46,7 @@ fun <T> DropdownMenu(
                 if (items.size > 1) ExposedDropdownMenuDefaults.TrailingIcon(expanded = expanded)
             },
             colors = TextFieldDefaults.textFieldColors(
```

Comment:
To fix the alignment of the `selectedItem` in the `ExposedDropdownMenu` component and center it, you can add `singleLine = true,` to ensure it is displayed on a single line.